register.py

```diff
@@ -0,0 +1,10 @@
+from wtforms import Form, BooleanField, StringField, validators
```

Comment:
according to PEP8, each import should be on its own line

Reply:
PEP8 is nto asking it in the CI and in an online tool. whene did you see it? can you point me please.
Check this URL: http://pep8online.com/checkresult